_docs_v7/Theory.md

```diff
@@ -87,6 +89,68 @@ Within the `EULER` solvers, we discretize the equations in space using a finite
 
 ---
 
+# Thermochemical Nonequilibrium Navier-Stokes #
+
+| Solver | Version | 
+| --- | --- |
+| `NEMO_NAVIER_STOKES`, `NEMO_RANS` | 7.0.0 |
```

Comment:
Is NEMO_RANS operational already?

Reply:
Last I talked to @WallyMaier it works in his branch, but it's not in develop. I think he is still doing V&V. Should I remove the NEMO_RANS ref here until it's in develop?

Reply:
The documentation and code branches should be in sync (develop -> develop, master -> master)